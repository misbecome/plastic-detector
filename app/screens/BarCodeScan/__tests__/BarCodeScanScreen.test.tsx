import React from "react";
import { create } from "react-test-renderer";

import BarCodeScanScreen from "../BarCodeScanScreen";

it("BarCodeScanScreen renders correctly", () => {
  const tree = create(<BarCodeScanScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
