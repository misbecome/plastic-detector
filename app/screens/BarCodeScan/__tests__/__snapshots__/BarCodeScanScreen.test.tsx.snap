// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarCodeScanScreen renders correctly 1`] = `
<Text.Primary>
  Requesting for camera permission
</Text.Primary>
`;
